README.md

```diff
@@ -223,6 +223,7 @@ Exposed as `eio` in the browser standalone build.
         (see [ws module](https://github.com/einaros/ws) api docs). Set to `false` to disable. (`true`)
         - `threshold` (`Number`): data is compressed only if the byte size is above this value. This option is ignored on the browser. (`1024`)
       - `extraHeaders` (`Object`): Headers that will be passed for each request to the server (via xhr-polling and via websockets). These values then can be used during handshake or for special proxies. Can only be used in Node.js client environment.
+      - `requestTimeout` (`Number`): Timeout for xhr-polling requests in milliseconds, defaults to no timeout.
```

Comment:
I am wondering whether we should define a large default timeout here as a best practice. Something like one minute should be long enough so that it does not interfere with existing users and short enough that it releases browser resources at some point.